src/state/lottery/getUserLotteryData.ts

```diff
@@ -19,7 +19,7 @@ const applyNodeDataToUserGraphResponse = (
         status: nodeRound.status,
         lotteryId: nodeRound.lotteryId.toString(),
         claimed: hasRoundBeenClaimed(ticketDataForRound.userTickets),
-        totalTickets: `${ticketDataForRound.userTickets.length}`,
+        totalTickets: `${ticketDataForRound.userTickets.length.toString()}`,
```

Comment:
This is expecting a string value, so calling `.toString()` on .length (it's rare that this data will ever be returned, hence why this hasn't come up)